src/client/studies/MCF/IntronAnalysis.js

```diff
@@ -52,45 +47,39 @@ class IntronAnalysis extends React.Component {
       if (this.verticals) {
         _.each(this.service.samples, () => bellCurveChartData.pop())
       }
+      if (this.subsetVerticals) {
```

Comment:
This removes 2 verticals at a time right? Even if the verticals don't exist?

It's probably better to keep a count of the verticals like you do in `MedianDistribution`.